browser/extensions/brave_component_loader.cc

```diff
@@ -108,6 +109,14 @@ void BraveComponentLoader::AddDefaultComponentExtensions(
       brave_webtorrent_path.Append(FILE_PATH_LITERAL("brave_webtorrent"));
     Add(IDR_BRAVE_WEBTORRENT, brave_webtorrent_path);
   }
+
+#if BUILDFLAG(BRAVE_WALLET_ENABLED)
+  // If brave://wallet has been loaded at least once, then load it again.
+  if (ExtensionPrefs::Get((content::BrowserContext *)profile_)->
```

Comment:
lint -> `content::BrowserContext *` -> `content::BrowserContext*` - but is this even needed? Is `profile_` a `void*` or something here?

Reply:
Hrm not sure why the cast was there, it was copied from within the same file. I fixed both spots though to not have the cast.